f5-distributed-cloud/README.md

```diff
@@ -0,0 +1,99 @@
+## Overview
+
+F5 Distributed Cloud (XC) Services provides a global cloud native platform where customers can deploy, manage, and secure their applications regardless of whether the application resides in a public cloud, in a private data center, or a colocation facility (see below). It provides a variety of both ADN and CDN services.
+
+![snapshot][1]
+
+Although the F5 XC console UI provides very good observability natively, many enterprises prefer to aggregate their telemetry from various sources and centralize visibility/analytics on the Datadog platform.  To this end, the F5 XC platform includes the Global Log Receiver.
+
+A F5 XC global log receiver can be configured to securely send logs to a Datadog HTTPS logging endpoint. Configuration is quite simple and can be done via the [F5 XC Console UI][6].
+
+In addition to the following configuration guidance. The F5 Distributed Cloud Services integration includes:
+
+- Dashboard - *Access Log Overview*
+- Saved View - *Including facets for commonly queried fields*
+
+![snapshot][2]
+
+## Setup
+
+You can configure global log streaming in either system namespace or in shared namespace. In case of configuring in shared namespace, you can configure to stream from either shared namespace or all namespaces or specific list of namespaces. If you are configuring in system namespace, you can only stream logs from system namespace.
+
+The example shown in this guide creates a global log receiver object in the Console in system namespace for streaming the logs to the external log collection system.
+
+Perform the following in the F5® Distributed Cloud Console:
+
+**Step 1: Start creating a global log receiver**
+
+- In the Console home page, select Shared Configuration service.
+- Select Manage > Global Log Receiver.
+- Select Global Log Receiver in case of Cloud and Edge Sites service.
+
+![snapshot][3]
```

Comment:
please see comment regarding images in the overview section.

Reply:
updated images will be in next pull request

Reply:
Hey Greg! Could you provide insight on what you mean by the next pull request? We're hoping to get everything squared away for the initial launch! We try to have everything inclusive in the first PR so future updates are minimal.